README.md

```diff
@@ -184,6 +184,9 @@ git@github.com:rosemutai/My-Portfolio.git
 - [ ] **making the porfolio responsive**<br/>
 - [ ] **Adding Animation**<br/>
 - [ ] **Making it dynamic**
+- [ ]  **Loom link explaining the project** https://www.loom.com/share/a837017526624df386ce0423e4832f76
```

Comment:
Excellent job documenting your project😉 It looks even easier to understand with the loom video😎😎 Let's make the readme look more professional by correcting this little mistake😉 

- [ ] It seems you are making a mistake for the header of this section😉 I think it should be `future features` rather than `key features` as you already have the `key features` section before and the contents also look more like that of the `future feature` section. Kindly make the necessary correction👍
- [ ] I think the checkbox for your readme which you intended to implement is not showing because of the space between the square brackets😉 this is making your documentation look a bit scattered, kindly see the image below and fix it👍
<img width="435" alt="yourReadme" src="https://user-images.githubusercontent.com/64643995/223181945-208c58de-c9dc-4833-afba-e2300317dfe4.png">

Don't forget that documenting our projects professionally is a [requirement](https://github.com/microverseinc/curriculum-html-css/blob/main/portfolio/7_mobile_menu.md) in this milestone of the project😉😉